synapseclient/models/file.py

```diff
@@ -0,0 +1,213 @@
+import asyncio
+from dataclasses import dataclass
+from typing import Dict, Union
+from opentelemetry import trace, context
+from synapseclient.models import AnnotationsValue, Annotations
+
+# import uuid
+
+from synapseclient.entity import File as Synapse_File
+from synapseclient import Synapse
+
+from typing import Optional, TYPE_CHECKING
+
+
+if TYPE_CHECKING:
+    from synapseclient.models import Folder, Project
+
+
+tracer = trace.get_tracer("synapseclient")
+
+
+@dataclass()
+class File:
```

Comment:
This might look odd with the auto documentation. You may want to document the class with attributes.